src/app.rs

```diff
@@ -457,6 +480,18 @@ impl eframe::App for BdiffApp {
             self.show_goto_modal(&goto_modal, ui, ctx);
         });
 
+        let overwrite_modal: Modal = Modal::new(ctx, "overwrite_modal");
+
+        if self.overwrite_modal.open {
+            self.overwrite_modal(&overwrite_modal);
+            overwrite_modal.open();
+        }
+
+        // Standard HexView input
```

Comment:
Since we already do this above if the goto modal isn't open, I think we want this logic to look like "if neither modal is open, self.handle_hex_view_input" rather than checking one at a time like this and handling input twice